src/Microsoft.TestPlatform.Common/Telemetry/TelemetryDataConstants.cs

```diff
@@ -68,6 +68,8 @@ public static class TelemetryDataConstants
 
         public static string ParallelEnabledDuringDiscovery = "VS.TestDiscovery.ParallelEnabled";
 
+        public static string ParallelEnabledDuringStartTestSession = "VS.StartTestSession.ParallelEnabled";
```

Comment:
Why are these fields are not constants marked with 'const' keyword?

Reply:
The existing fields were already marked as `static` so I marked mine `static` too for the sake of consistency.